syntax = "proto3";

package absolute_skew_deviation;

message AbsoluteSkewDeviationRequest {
    string symbol = 1;
    int32 lookback = 2;
}

message AbsoluteSkewDeviationResponse {
    bytes series = 1;
}

service AbsoluteSkewDeviation {
    rpc get_absolute_skew_deviation (AbsoluteSkewDeviationRequest) returns (AbsoluteSkewDeviationResponse);
}